frontend/src/components/AgencyProfile/AgencySideBar.jsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { Component, useEffect, useState } from "react";
+import "./AgencyProfile.css";
+import { NavHashLink } from "react-router-hash-link";
+
+/**
+ * Functional component for Agency Side Bar
+ *
+ * @param {*} getScrollPositions function that returns page offset of agency categories
+ * @param {string} id unique id of agency
+ * @param {*} ScrollTo function that scrolls page to given element
+ * @returns {*} Agency Side Bar Component
+ */
+function AgencySideBar({ getScrollPositions, id, positions, ScrollTo }) {
+  const [scrollPos, setScrollPos] = useState("");
```

Comment:
`scrollPos` is never used - consider removing

Reply:
scrollPos is used to re-render the function when the scroll position changes based off the different agency categories.

Reply:
`setScrollPos` is called, but `scrollPos` is never referred to anywhere. Are you just changing the state so that it re-renders? If so, maybe add a quick comment saying that it's just used to trigger renders